locales/fr/translation.json

```
@@ -288,49 +307,53 @@
   "Hide All": "Tout cacher",
   "Hide Password": "Cacher le mot de passe",
   "high": "haut",
-  "high-pa": "forte",
   "High": "Haut",
+  "high-pa": "forte",
   "Hmm, {{field}} usually comes after birthday.": "Hmm, {{field}} est habituellement après la date de naissance.",
   "Hmm, diagnosis date usually comes after birthday": "Hmm, la date du diagnostic devrait être après la date de naissance.",
+  "Hmm, this date doesn't look right": "",
```

```suggestion
  "Hmm, this date doesn't look right": "Hmm, cette date ne semble pas correcte",
```
And suppress the next line + change app/core/personutils.js:153
The simple quote used here (’) is the french version, not the en-US one (').